.gitignore

```diff
@@ -1,5 +1,5 @@
 .DS_Store
 *.log
-dist
-lib/**.js
+lib
```

Comment:
I don't get it. Why push "compiled" files on github? The lib folder will anyway be pushed on `npm` in case of npm publish (e.g: for [redux](https://github.com/reactjs/redux/blob/master/.gitignore#L4-L5))

IMHO it also bring confusion to new users ("Where should I code? Where is the source?"). But np if you prefer keep it, for consistency across repos.